apps/server/src/modules/tool/controller/mapper/external-tool-response.mapper.ts

```diff
@@ -61,7 +63,7 @@ export class ExternalToolResponseMapper {
 		);
 
 		return new ExternalToolResponse({
-			id: externalToolDO.id || '',
+			id: externalToolDO.id ?? '',
```

Comment:
What is the reason to change it from || to ??

Reply:
The || operator returns the first truthy value or the second value if the first is falsy (including 0, '', false, null, undefined, and NaN). The ?? operator returns the first value unless it's null or undefined, in which case it returns the second value.
So || is enough the other things should and can not happen